### PR TITLE
[GG-2810] Fixes lighthouse

### PR DESCRIPTION
**Pull request by sergioazevedo**
## Description

This PR aims to fix the lighthouse check.
In the end, I could not figure out why exactly this check is failing. 
I managed to run the `puppeteer-script.js` locally and everything was working fine, but it was not working properly when running via GH Actions, which made my life very difficult.
Lastly, following Niklas's suggestion, I introduce a `sleep` in puppeteer-script.js and to my surprise, this `sleep` fixed the lighthouse check. 
Niklas and I looked for a better and more elegant solution than the `sleep`, we tried a bunch of different puppeteer methods like: `page.waitForFunction, page.evaluate, waitUntil` but without luck 😞. So in the end I gave up and decided to open this PR with the old fashion `sleep`.

## Checklist

- [ ] :green_book: all commit messages follow the [conventional commits](https://conventionalcommits.org/) standard
- [ ] :nail_care: SASS files are compiled
- [ ] :arrow_left: changes are compatible with RTL direction
- [ ] :wheelchair: changes are accessible
- [ ] :memo: changes are tested in Chrome, Firefox, Safari and Edge
- [ ] :iphone: changes are responsive and tested in mobile
- [ ] :+1: PR is approved by @zendesk/vikings

### CODE REVIEW

File: .github/workflows/puppeteer-script.js [hunk 1]
```diff
@@ -4,7 +4,6 @@ module.exports = async (browser, context) => {
     await page.goto('https://' + process.env.subdomain + '.zendesk.com/auth/v2/login');
     await page.type('#user_email', process.env.end_user_email);
     await page.type('#user_password', process.env.end_user_password);
-    const navigationPromise = page.waitForNavigation();
     await page.click('[name="commit"]');
-    await navigationPromise;
+    await setTimeout(() => (console.log("login in")), 100);
```
Comment (jbdietrich) [inline .github/workflows/puppeteer-script.js:8]:
```suggestion
    await setTimeout(() => (console.log("logging in")), 100);
```

**Review by jbdietrich — APPROVED** [review]
LGTM, but I am a noob.

**Review by jbdietrich — APPROVED** [review]

**Review by JakubKopys — APPROVED** [review]
not a fan of that, but I guess if it works... I guess we can leave a comment that explains why its there (because it failed and we have no clue why it fixes it but it does 🤷)

**Pull request merged**